DataManagementSystem/Agent/RequestOperations/RemoveFile.py

```diff
@@ -55,7 +55,7 @@ def __init__( self, operation = None, csPath = None ):
     gMonitor.registerActivity( "RemoveFileFail", "Failed file removals",
                                "RequestExecutingAgent", "Files/min", gMonitor.OP_SUM )
     # # re pattern for not existing files
-    self.reNotExisting = re.compile( "(no|not) such file or directory", re.IGNORECASE )
+    self.reNotExisting = re.compile( "(no|not) such file.*", re.IGNORECASE )
```

Comment:
I am far from an expert of the re module (actually I know nothing of it), but I thought this was the default ?

In [1]: import re
In [2]: regex = re.compile('myString')
In [3]: regex
Out[3]: re.compile(r'myString')